SPECS/xenopsd.spec

```diff
@@ -225,7 +225,9 @@ esac
 %post xc-cov
 case $1 in
   1) # install
+    rm -f %{_sbindir}/xenopsd-xc
     ln -s %{_sbindir}/xenopsd-xc.cov %{_sbindir}/xenopsd-xc
+    rm -f %{_libexecdir}/%{name}/set-domain-uuid
```

Comment:
You should use `ln -sf` if you want to override. This is cleaner than having an `rm -f` additionally.